Makefile

```diff
@@ -74,6 +74,9 @@ all: lint install
 install: go.sum
 		go install $(BUILD_FLAGS) ./cmd/seid
 
+install-mock-balances: go.sum
+		go install -tags "$(build_tags) mock_balances" -ldflags '$(ldflags)' ./cmd/seid
```

Comment:
shouldn't we set build tags in all "install" commands? And if so, we won't need the extra command for install-mock-balances, because `BUILD_TAGS=mock_balances make install` will do

Reply:
we do, those are set via `BUILD_FLAGS`, but that is set using env vars prior to the targets, so that's why i modified it to be this way. (I can clean it up to be nicer)